src/libraries/Microsoft.PowerFx.Connectors/Internal/TabularDType.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using Microsoft.PowerFx.Core.Types;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Connectors
+{
+    // Used in TabularIRVisitor
+    internal class TabularDType : DType
```

Comment:
>TabularDType [](http://example.com/codeflow?start=19&length=12)

This is probably fine now; but eventually it should be using DType.AssociatedDAtaSources.

for example, we'll need to capture delegation capabilities.

Reply:
For future, yes
Mostly needed in TabularIRVisitor

Reply:
For PA/DV, tabular is specified via an existing table DType , but with AssociatedDAtaSources set to extra info